AirGradient-ESPHome/airgradient-diy-pro.yaml

```diff
@@ -0,0 +1,121 @@
+---
+# Adapted from: https://github.com/m-reiner/airgradient-esphome-prometheus
+esphome:
+  name: !secret name
+  platform: ESP8266
+  board: d1_mini
+
+logger:
+
+api:
+  password: !secret api_password
```

Comment:
These would need updating now, to use the new API key setup, right?

Reply:
I plan to address this within the next month or two, including support for their new Open Air module.

Thanks for pointing this out.